ext/web/06_streams.js

```diff
@@ -742,14 +742,15 @@ function readableStreamForRid(rid, autoClose = true) {
           return;
         }
 
-        const bytesRead = await core.read(rid, v);
+        // deno-lint-ignore prefer-primordials
+        const [bytesRead, newView] = await core.readDetaching(rid, v);
```

Comment:
Please use object destructuring patterns. Array patterns call `Array.prototype[Symbol.iterator]`.

```suggestion
        const { 0: bytesRead, 1: newView } = await core.readDetaching(rid, v);
```